setup.py

```diff
@@ -27,7 +27,8 @@
     'python-memcached',
     'django-cachalot',
     'geoip',
-    'django-webpack-loader'
+    'django-webpack-loader',
+    'pip'
```

Comment:
This is still in the `project_template` so is still needed but it could be moved to being done manually after `start_cms_project.py` or adding a flag to that script like `--webpack` that adds it to the `requirements.txt` and `pip installs` it after